arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -672,7 +673,8 @@
             "modelAsString": false
           }
         }
-      }
+      },
+      "description": "The parameters used to create storage account."
```

Comment:
Can you add a "the" in there, so it is "The parameters used to create the storage account." ?